packages/core/src/utils/dom.ts

```
@@ -377,27 +391,16 @@ export function walkTextNodes(
     const node = nodes[i]
     const nodeType = node.nodeType
 
-    if (nodeType === 3) {
+    if (nodeType === NodeType.TEXT_NODE) {
       // 匹配到 text node ，执行函数
       handler(node, elem)
-    } else if (nodeType === 1 || nodeType === 9 || nodeType === 11) {
+    } else if (nodeType === NodeType.ELEMENT_NODE || nodeType === NodeType.DOCUMENT_NODE || nodeType === NodeType.DOCUMENT_FRAGMENT_NODE) {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)